pkg/handler/handler.go

```diff
@@ -688,9 +690,18 @@ func (h handler) lastoperation(w http.ResponseWriter, r *http.Request, params ma
 		return
 	}
 
+	// we have a binding job
+	if strings.Index(r.URL.Path, "/service_bindings/") > 0 {
+		bindingUUID := uuid.Parse(params["binding_uuid"])
+		if bindingUUID == nil {
+			writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: "invalid binding_uuid"})
+			return
+		}
+	}
+
 	req := broker.LastOperationRequest{}
 
-	// operation is rqeuired
+	// operation is required
```

Comment:
Speaking of this comment, the code below actually makes it optional. <shrug> It just logs a warning but continues its lookup using just the service ID. I think it would be better to start enforcing what this comment says, and make it required, if not for the service last operation, at least for this case of a binding last operation.

If we let it continue without the ``operation``, I'm not sure how it would behave. Maybe it would just fail to lookup the state? Or would it grab some other ``JobState`` for the same service ID, but not necessarily the right one? Either way, it's better to not let it get that far.

Reply:
Great point.

Reply:
It returns an error

```
< HTTP/1.1 400 Bad Request                          
< Content-Type: application/json                    
< Date: Tue, 23 Jan 2018 16:33:31 GMT               
< Content-Length: 124                               
< Set-Cookie: d05351dd24520dce41f310fcfc112523=2808fe5497bbaeea73011198cb5cc014; path=/; HttpOnly; Secure                                                                                                          
<                                                   
{                                                   
  "description": "operation not supplied for a last_operation with instance_uuid [3aae62b1-f990-43de-89b6-25ad2c69e4e1]"                                                                                            
} 
```